### PR TITLE
Update dependencies.sh

### PR DESCRIPTION
**Pull request by baharmoosavi**
I can't access https://git.code.sf.net/p/ngspice/ngspice , but it is possible with git:// ...

### CODE REVIEW

**Review by baharmoosavi — COMMENTED** [review]
I can't access https://git.code.sf.net/p/ngspice/ngspice , but it is possible with git:// ...

**Comment by chetanyagoyal** [comment]
can you enable workflows on your repo to check if they pass @baharmoosavi

File: dependencies.sh [hunk 1]
```diff
@@ -212,7 +212,7 @@ source ~/.bashrc
 if cat /etc/os-release | grep "ubuntu" >> /dev/null
 then
  apt install bison flex libx11-dev libx11-6 libxaw7-dev libreadline6-dev autoconf libtool automake -y
- git clone https://git.code.sf.net/p/ngspice/ngspice
+ git clone git://git.code.sf.net/p/ngspice/ngspice
```
Comment (chetanyagoyal) [inline dependencies.sh:215]: can you make a similar change in the Dockerfile to see if it passes in the CI

**Review by chetanyagoyal — CHANGES_REQUESTED** [review]

**Comment by chetanyagoyal** [comment]
closing this soon as it seems like a non-issue. @baharmoosavi are you planning to work on this further?

**Pull request closed without merging**